vendor/github.com/openshift/api/config/v1/types_cluster_version.go

```diff
@@ -245,17 +245,22 @@ const (
 	// supplies Operator Lifecycle Manager (OLM) users with default catalogs of
 	// "optional" operators.
 	ClusterVersionCapabilityMarketplace ClusterVersionCapability = "marketplace"
+
+	// ClusterVersionCapabilityConsole manages the Console operator which
+	// installs and maintains the web console.
+	ClusterVersionCapabilityConsole ClusterVersionCapability = "console"
```

Comment:
This needs to be rerolled to pick up openshift/api#1240's pivot to `Console`.  Or we can move to #812, since that has both `Console` and `Insights`.